.travis.yml

```diff
@@ -9,8 +9,9 @@ dist: xenial
 
 
 before_install:
-    - wget 'https://github.com/cylc/cylc/archive/master.tar.gz' -O '/tmp/cylc-master.tar.gz'
+    - wget 'https://github.com/cylc/cylc-flow/archive/master.tar.gz' -O '/tmp/cylc-master.tar.gz'
     - tar -xvf '/tmp/cylc-master.tar.gz' -C "${HOME}"
+    - mv "${HOME}/cylc-flow-master" "${HOME}/cylc-master"
```

Comment:
This seems inelegant. Alternative suggestions gladly welcomed.

Reply:
OK for now. Maybe raise an issue later to address this (and any other instances where we reference *cylc*.